specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -642,10 +642,31 @@
         ],
         "parameters": [
           {
-            "$ref": "#/parameters/CreateLivenessWithVerifySessionContent.Parameters"
+            "name": "contentType",
```

Comment:
is this something that should've always had contentType, or should contentType

Reply:
it is from the `CreateLivenessWithVerifySessionContent` model.

Reply:
Maybe we need to refactor this? I assume
https://github.com/Azure/azure-rest-api-specs/blob/232f7aac9acd8b587cc8ba64cf7016095b504d7b/specification/ai/Face/models.session.tsp#L269-L281

Does not want `content-type` to be in formData

Reply:
resolved.